sdk/keyvault/azure-keyvault-secrets/tests/_async_test_case.py

```diff
@@ -0,0 +1,42 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import functools
+import os
+
+import pytest
+from azure.keyvault.secrets._shared.client_base import DEFAULT_VERSION
+from devtools_testutils import (AzureRecordedTestCase,
+                                EnvironmentVariableLoader, is_live)
+
+KeyVaultSecretsPreparer = functools.partial(EnvironmentVariableLoader, "keyvault", azure_keyvault_url="https://vaultname.vault.azure.net")
+
+
+class AsyncSecretsTestCaseClientPrepaper(AzureRecordedTestCase):
```

Comment:
We can probably abbreviate this a smidge since there's one less preparer in the mix
```suggestion
class AsyncSecretClientPreparer(AzureRecordedTestCase):
```